### DISCUSSION TITLE
Preferences never saved

### DISCUSSION THREAD

**Discussion by jonnycjd** [small-cactus/M.I.L.E.S#21] [Q&A] [answered]
Hello - after entering my OpenAI API key, preferred city and metric, I get prompted to download files then say the "This is a test" speech.  After the test, I am prompted to restart Miles to save preferences. However it takes me back to the step of entering API key etc. Please advise? 
![image](https://github.com/small-cactus/M.I.L.E.S/assets/170960912/ff5de6dd-4a7a-42ac-b972-893678783686)

**Comment by small-cactus** [comment, marked as answer]
Did you fill out the other options? You need to at least type something in the fields for it to save.

**Reply by jonnycjd** [reply]
This was it - thanks for the prompt reply

**Reply by small-cactus** [reply]
no problem, glad I could help.
